### ISSUE TITLE
Use JSON log format for node production environments

### ISSUE THREAD

**Issue by anjmao** [mysteriumnetwork/node#1384]
Currently we output logs in text format in both development and production environments. I propose that logs should use json format for prod environments.

JSON logs would be easer to process as there are fields like level, time, caller, message.
```
{"level":"debug","time":"2019-10-31T11:37:12.576127+02:00","caller":"/core/discovery/api/fetcher.go:100       ","time":"2019-10-31T11:37:12.57615+02:00","caller":"/core/discovery/api/fetcher.go:100       ","time":"2019-10-31T11:37:12.576159+02:00","message":"Proposals fetched: 1"}
```

Coloured text logs are nice for development but it would be hard to parse if we want to have some tools for nodes logs aggregation and processing.
```
2019-10-31T11:38:55.443 DBG core/discovery/api/fetcher.go:100        > Proposals fetched: 1
```

**Comment by eugenegoncharuk** [comment]
@mdomasevicius Can You, please, check?
